com.unity.netcode.gameobjects/Tests/Runtime/Metrics/PacketLossMetricsTests.cs

```diff
@@ -27,6 +28,7 @@ public PacketLossMetricsTests()
         protected override void OnServerAndClientsCreated()
         {
             var clientTransport = (UnityTransport)m_ClientNetworkManagers[0].NetworkConfig.NetworkTransport;
+            clientTransport.DebugSimulatorRandomSeed = 4; // Determined through trial and error.
```

Comment:
Uhhh can you explain this trial and error a little bit

Reply:
Oops, sorry - I should have read the PR description

Reply:
I think it might be worth including a comment about why the seed is important, future updates to UTP could change the call pattern to the RNG again and break this test and some poor developer is going to have to retrace your steps